### ISSUE TITLE
Cannot use TensorBoard plugin at all: There’s no dashboard by the name of “pytorch_profiler.”

### ISSUE THREAD

**Issue by fzyzcjy** [pytorch/kineto#705]
![image](https://user-images.githubusercontent.com/5236035/206367890-19491d50-c5d1-46e5-b708-86af67cc1e9b.png)
![image](https://user-images.githubusercontent.com/5236035/206367908-ceaf8b9a-18f3-4d00-977f-7d8dbe35fab2.png)

```
>>> import tensorboard
>>> tensorboard.__version__
'2.11.0'
>>> import torch_tb_profiler
>>> torch_tb_profiler.__version__
'0.4.0'

**Comment by robieta** [comment]
Are there any logs from the TensorBoard process?

**Comment by fzyzcjy** [comment]
Hi, I have fixed the problem: `tensorboard` launches v1.12 (though `which tensorboard` points to the correct 2.11)

Thanks for your reply!
